README.md

```
@@ -105,6 +115,53 @@ google-takeout-metadata --batch --localtime --clean-sidecars "/chemin/vers/dossi
+
+**Nouvelle fonctionnalité** : Organisation automatique des fichiers selon leur statut dans Google Takeout.
+
+```bash
+# Activer l'organisation automatique
+google-takeout-metadata --organize-files /chemin/vers/le/dossier
+
+# Combiner avec d'autres options
+google-takeout-metadata --batch --organize-files --localtime /chemin/vers/le/dossier
+```
+
+### 📁 Fonctionnement:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)